net/scripts/colo-node-onacquire.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# These variable must be set before the main body is called
+SOLANA_LOCK_FILE="${SOLANA_LOCK_FILE:?}"
+INSTANCE_NAME="${INSTANCE_NAME:?}"
+SSH_AUTHORIZED_KEYS="${SSH_AUTHORIZED_KEYS:?}"
+SSH_PRIVATE_KEY_TEXT="${SSH_PRIVATE_KEY_TEXT:?}"
+SSH_PUBLIC_KEY_TEXT="${SSH_PUBLIC_KEY_TEXT:?}"
+NETWORK_INFO="${NETWORK_INFO:-"Network info unavailable"}"
+CREATION_INFO="${CREATION_INFO:-"Creation info unavailable"}"
+
+if [ ! -f "$SOLANA_LOCK_FILE" ]; then
+  exec 9>>"$SOLANA_LOCK_FILE"
+  flock -x -n 9 || ( echo "Failed to acquire lock!" 1>&2 && exit 1 )
+  ( [ -n "$SOLANA_USER" ] && {
+    echo "export SOLANA_LOCK_USER=$SOLANA_USER"
+    echo "export SOLANA_LOCK_INSTANCENAME=$INSTANCE_NAME"
+    echo "[ -v SSH_TTY -a -f \"${HOME}/.solana-motd\" ] && cat \"${HOME}/.solana-motd\" 1>&2"
+  } >&9 ) || ( rm "$SOLANA_LOCK_FILE" && echo "SOLANA_USER undefined" 1>&2 && false )
+  exec 9>&-
+  cat > /solana-scratch/id_ecdsa <<EOK
```

Comment:
EOK/EOAK/EOM, is there any significance to the different words used here?  If no I'd prefer just a consistent `EOF` (or whatever), so my brain doesn't need to wonder why they aren't all the same

Reply:
Key/AuthorizedKeys/MOTD, respectively.  At one point this block was inside an `<<EOF`, hence the divergence.  They can be renamed now